app/src/main/res/layout/fragment_article.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.recyclerview.widget.RecyclerView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I think you overcomplicated this screen, there is no need for `RecyclerView`. Just one scrollable `TextView` should be sufficient, and you need to combine all text to one `SpannableString`, then that's it.